src/connection/tcp.c

```diff
@@ -257,7 +258,7 @@ css_hostname_to_ip (const char *host, unsigned char *ip_addr)
       int herr;
       char buf[1024];
 
-      if (gethostbyname_r (host, &hent, buf, sizeof (buf), &herr) == NULL)
+      if (gethostbyname_r_uhost (host, &hent, buf, sizeof (buf), &herr) == NULL)
```

Comment:
If host_conf_Use is 1, gethostbyname_r_uhost() cannot return NULL. gethostbyname_r_uhost () Please check again.

Reply:
Thank you. I will check it again.